README.md

```diff
@@ -226,9 +228,9 @@ e.g
 ```
  ilib-assemble output-dir --legacyilib --ilibPath ~/Source/develop/ --ilibincPath src/ilib-all-inc.js -l "af-ZA,am-ET,ko-KR"
 ```
+- Add the option `customLocalePath` to provide a possibility to overwrite the locale data. Basically, the locale data is based on the CLDR.  
+ If you want to have it to return a different value, Prepare the JSON locale data to override the same locale hierarchy and use the option when running the tool.
```

Comment:
- Add the option `customLocalePath` to override the locale data with custom data. The open-source locale data is based only on the CLDR. If your company wants override certain settings, it can create custom JSON locale data in the same format and the same locale hierarchy as the ilib files, and any data in the custom files will override the parallel data from CLDR.

Reply:
I update it.